src/components/MultiHopTrade/components/TradeInput/TradeInput.tsx

```diff
@@ -184,8 +184,6 @@ export const TradeInput = ({ isCompact }: TradeInputProps) => {
       case !!quoteResponseError:
         return getQuoteRequestErrorTranslation(quoteResponseError)
       case !!tradeQuoteError:
-        // this should never occur because users shouldn't be able to select an errored quote
-        // but just in case
```

Comment:
This is not true, and this case is indeed hit.

We _do_ in fact allow non-actionable quotes to be selected, just not acted upon.

Reply:
🧠